extensions/ql-vscode/package.json

```diff
@@ -114,7 +114,7 @@
           "scope": "machine",
           "type": "string",
           "default": "",
-          "description": "Path to the CodeQL executable that should be used by the CodeQL extension. The executable is named `codeql` on Linux/Mac and `codeql.exe` on Windows. This overrides all other CodeQL CLI settings."
```

Comment:
Maybe add something like "If empty, use the extension's default CLI."

Reply:
How about "If empty, the extension will look for a CodeQL executable on your shell PATH, or if CodeQL is not on your PATH, download and manage its own CodeQL executable."

Reply:
That's even better.